### PR TITLE
Update asuswrt.markdown

### PR DESCRIPTION
**Pull request by jwater7**
Reflect new released changes that disables sensors by default.  Added example and better description of sensors.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19736

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/asuswrt.markdown [hunk 1]
```diff
@@ -26,6 +26,25 @@ asuswrt:
   username: YOUR_ADMIN_USERNAME
 ```
 
+To enable ASUSWRT sensors as part of your installation, reference the following example configuration:
```
Comment (fabaff) [inline source/_components/asuswrt.markdown]: Move this sample configuration to the end of the file and add a linkable title please.

File: source/_components/asuswrt.markdown [hunk 2]
```diff
@@ -67,7 +86,7 @@ sensors:
   description: List of enabled sensors
   required: false
   type: list
-  default: all (`upload`, `download`, `upload_speed`, `download_speed`)
+  default: None
```
Comment (fabaff) [inline source/_components/asuswrt.markdown]: Can be removed if there is no default.

**Review by fabaff — DISMISSED** [review]

File: source/_components/asuswrt.markdown (hunk 1 above)
Comment (jwater7) [reply source/_components/asuswrt.markdown]: Done

File: source/_components/asuswrt.markdown (hunk 2 above)
Comment (jwater7) [reply source/_components/asuswrt.markdown]: Done

**Comment by jwater7** [comment]
Updated for requested changes, should be good to go - thanks

**Comment by jwater7** [comment]
Any additional changes or anything blocking in order to merge?

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
I think this PR is okay now! 👍

**Comment by klaasnicolaas** [comment]
@fabaff Can we merge this PR?

**Review by frenck — DISMISSED** [review]

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @jwater7! 👍

**Pull request merged**